modules/ossm-about-concepts-kiali.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// about/ossm-about-concepts-assembly.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-concepts-kiali_{context}"]
+= {SMProductName} and Kiali
+
+//Kiali attributes are in the works, and possible {KialiProduct} will refer to Kiali provided by Red Hat, and not Kiali Operator provided by Red Hat.
+//Kiali attributes are outside the scope of this PR.
+
+Kiali is composed of three parts:
+
+* {KialiProduct}
+* Kiali Server
+* {SMPlugin}
+
+Working together, they form the user interface (UI) for {SMProduct}. Kiali provides visibility into your service mesh by showing you the microservices in your service mesh, and how they are connected.
+
+Kiali provides observability into the Service Mesh running on {ocp-product-title}. Kiali helps you define, validate, and observe your Istio service mesh. It helps you to understand the structure of your service mesh by inferring the topology, and also provides information about the health of your service mesh.
+
+Kiali provides an interactive graph view of your mesh namespaces in near real time that provides visibility into features like circuit breakers, request rates, latency, and even graphs of traffic flows. Kiali offers insights about components at different levels, from Applications to Services and Workloads, and can display the interactions with contextual information and charts on the selected graph node or edge.
+
+Kiali also provides the ability to validate your Istio configurations, such as gateways, destination rules, virtual services, mesh policies, and more. Kiali provides detailed metrics, and a basic Grafana integration is available for advanced queries. Distributed tracing is provided by integrating {TempoName} and {OTELName} into the Kiali console.
+
+[id="kiali-architecture_{context}"]
+== Kiali architecture
+
+Kiali is based on the open source link:https://kiali.io/[Kiali project]. Kiali is composed of two components: the Kiali Server and the Kiali console.
```

Comment:
This somewhat contradicts the above that states that Kiali is made up of three components, the server, the console and the openshift plugin (ossmc).